### PR TITLE
Made create table more generic.  Added create_materialized_view and create_materialized_view_log

### PR DESCRIPTION
**Pull request by madams51703**

### CODE REVIEW

File: plsql/PlSqlParser.g4 [hunk 1]
```diff
@@ -820,6 +822,84 @@ maxsize_clause
...
     ;
 
+build_clause
+    : BUILD (IMMEDIATE | DEFERRED)
+    ;
+
+parallel_clause
+    : ( NOPARALLEL
+      | PARALLEL parallel_count=UNSIGNED_INTEGER
+      )
```
Comment (KvanTTT) [inline plsql/PlSqlParser.g4]: Useless parenthesis.

File: plsql/PlSqlParser.g4 [hunk 2]
```diff
@@ -935,6 +967,67 @@ size_clause
...
+
+table_compression
+    : (COMPRESS
+        ( BASIC
+        | FOR ( OLTP
+              | (QUERY | ARCHIVE) (LOW | HIGH)?
+              )
+        )?
+      | NOCOMPRESS
+      )
```
Comment (KvanTTT) [inline plsql/PlSqlParser.g4]: Useless parenthesis.

File: plsql/PlSqlParser.g4 [hunk 3]
```diff
@@ -935,6 +967,67 @@ size_clause
...
+    ;
+
+physical_attributes_clause
+    :   (PCTFREE pctfree=UNSIGNED_INTEGER
```
Comment (KvanTTT) [inline plsql/PlSqlParser.g4]: Please use one space after the colon.

File: plsql/PlSqlParser.g4 [hunk 4]
```diff
@@ -935,6 +967,67 @@ size_clause
...
+        | PCTUSED pctused=UNSIGNED_INTEGER
+        | INITRANS inittrans=UNSIGNED_INTEGER
+        | storage_clause
+        )+
+    ;
+
+storage_clause
+    :  STORAGE '('
```
Comment (KvanTTT) [inline plsql/PlSqlParser.g4]: One space after colon.

File: plsql/PlSqlParser.g4 [hunk 5]
```diff
@@ -935,6 +967,67 @@ size_clause
...
+
+segment_attributes_clause
+    : ( physical_attributes_clause
+      | TABLESPACE tablespace_name=id_expression
+      | logging_clause
+      )+
+    ;
+
+physical_properties
+    : deferred_segment_creation?  segment_attributes_clause table_compression?
```
Comment (KvanTTT) [inline plsql/PlSqlParser.g4]: One space.

**Comment by madams51703** [comment]
Fixed formatting.  I don't see the extra space in the last comment.  Does
it look ok now?

**Comment by KvanTTT** [comment]
Ok, in general not so important.

**Pull request merged**